src/lib.rs

```diff
@@ -39,13 +51,12 @@ fn initial_grab_cursor(mut windows: ResMut<Windows>) {
 }
 
 /// Spawns the `Camera3dBundle` to be controlled
-fn setup_player(commands: &mut Commands) {
-    commands
-        .spawn(Camera3dBundle {
-            transform: Transform::from_translation(Vec3::new(0., 2., 0.)),
+fn setup_player(mut commands: Commands) {
+    commands.spawn_bundle(PerspectiveCameraBundle {
+            transform: Transform::from_xyz(-2.0, 72.0, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
             ..Default::default()
         })
-        .with(FlyCam);
+        .with::<FlyCam>();
```

Comment:
I ran your changes locally and get a build error here:
```
error[E0599]: no method named `with` found for struct `EntityCommands<'_, '_>` in the current scope
  --> src\lib.rs:59:10
   |
59 |         .with::<FlyCam>();
   |          ^^^^ method not found in `EntityCommands<'_, '_>`
```
Should be?:
```
.insert(FlyCam);
```
(Source: https://bevyengine.org/news/bevy-0-5/#entityref-entitymut)